org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/ParameterizedExpressionsTestLanguageGrammarAccess.java

```diff
@@ -591,37 +596,37 @@
 		private final Assignment cRhsAssignment_1_1_1 = (Assignment)cGroup_1_1.eContents().get(1);
 		private final RuleCall cRhsAssignmentExpressionParserRuleCall_1_1_1_0 = (RuleCall)cRhsAssignment_1_1_1.eContents().get(0);
 		
-		//AssignmentExpression <In, Yield Expression:
-		//	<Yield> YieldExpression<In> | RelationalExpression<In,Yield> (=> ({AssignmentExpression.lhs=current} op='=')
+		//AssignmentExpression <In,YieldExpression:
+		//	< Yield > YieldExpression<In> | RelationalExpression<In,Yield> (=> ({AssignmentExpression . lhs= current} op='=')
 		//	rhs=AssignmentExpression<In,Yield>)?;
 		@Override public ParserRule getRule() { return rule; }
 		
-		//<Yield> YieldExpression<In> | RelationalExpression<In,Yield> (=> ({AssignmentExpression.lhs=current} op='=')
+		//< Yield > YieldExpression<In> | RelationalExpression<In,Yield> (=> ({AssignmentExpression . lhs= current} op='=')
 		//rhs=AssignmentExpression<In,Yield>)?
 		public Alternatives getAlternatives() { return cAlternatives; }
 		
-		//<Yield> YieldExpression<In>
+		//< Yield > YieldExpression<In>
```

Comment:
@ArneDeutsch Missing formatting config for guard conditions?  There are unexpected spaces here inside the `< Yield >`

Reply:
Should be fixed with https://github.com/eclipse/xtext-core/issues/1489